_includes/layout/base/wallets-selector.html

```diff
@@ -0,0 +1,561 @@
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </div>
+            </div>
+          </div>
+          <div class="important-checkbox-row">
+            <label class="checkbox-wrapper">
+              <input type="checkbox" name="features" value="segwit" class="checkbox js-wallet-selector" />
+              <div class="checkbox-text">Segwit</div>
+            </label>
+            <div class="sidebar-selector-info tooltip-trigger">
+              <img src="/img/icons/info-alt.svg" alt="info">
+              <div class="wallet-selector-tooltip tooltip">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </div>
+            </div>
+          </div>
+        </div>
+        <h3 class="sidebar-selector-title">Need some help?</h3>
+        <p class="sidebar-selector-text">You can find more information and help on the resources and community pages or on the Wiki FAQ</p>
+      </div>
+
+      <div class="wallet-selector-content">
+        <div class="wallet-table-wrapper">        
+          <div class="wallet-table">        
+            <div class="wallet-table-row">
+              <div class="wallet-table-head">
+                <p class="wallet-table-title"><span id="selectedOs"></span> Wallets</p>
+              </div>
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">Control</p>
+              </div>
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">Validation</p>
+              </div>
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">Transparency</p>
+              </div>
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">Environment</p>
+              </div>
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">Fees</p>
+              </div>
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">Privacy</p>
+              </div>
+            </div>
+          
+            {% for wallet in site.wallets %}
+              {% for platform in wallet.platform %}
+                {% for os in platform.os %}
+                  <a
+                    href="{% if platform.name == os.name %}/{{page.lang}}/wallets/{{platform.name}}/{{wallet.id}}/{% else %}/{{page.lang}}/wallets/{{platform.name}}/{{os.name}}/{{wallet.id}}/{% endif %}"
+                    class="wallet-table-row wallet-link"
+                    data-categories="{{wallet.user}} {% for check in os.check %}{% assign checkValue = check[1] %}{% assign checkName = check[0] %}{% if checkValue contains 'checkgood' or checkValue contains 'checkpass' %}{{checkName}} {% endif %}{% endfor %}{{os.name}}"
```

Comment:
@alexcherman 

Let's show all wallets for the `experienced` user type, but continue to only show wallets with the `new` value for the `new` user type.

Reply:
ok, I'll do it

Reply:
Thank you.